src/picongpu/include/particles/InitFunctors.hpp

```diff
@@ -114,25 +116,40 @@ struct CreateDensity
                                destination and source, \see src/picongpu/include/particles/manipulators
  * @tparam T_SrcSpeciesType source species
  * @tparam T_DestSpeciesType destination species
+ * @tparam T_Filter picongpu::particles::filer, particle filter type to select particles
```

Comment:
This should mark that the parameter must follow the interface `picongpu::particles::filer` the comma is separating the description from the type.